easybuild/easyconfigs/f/FreeFEM/FreeFEM-4.5-foss-2019b.eb

```
@@ -0,0 +1,30 @@
+name = 'FreeFEM'
+version = '4.5'
+
+homepage = 'https://freefem.org'
+description = """FreeFEM offers a fast interpolation algorithm and a language for the manipulation
+ of data on multiple meshes."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = ['https://github.com/FreeFem/FreeFem-sources/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['5b2d4125c312da8fbedd49a72e742f18f35e0ae100c82fb493067dfad5d51432']
+
+builddependencies = [
+    ('Autotools', '20180311'),
+    ('flex', '2.6.4'),
+    ('Bison', '3.3.2'),
+    ('CMake', '3.15.3'),
+]
+
+dependencies = [
+    ('GSL', '2.6'),
+    ('HDF5', '1.10.5'),
+    ('SuiteSparse', '5.6.0', '-METIS-5.1.0'),
+    ('freeglut', '3.2.1'),
+    ('PETSc', '3.12.4'),
+    ('SLEPc', '3.12.2'),
+]
```

We can easily reduce the amount of downloaded deps by adding the following from EB
```suggestion
    ('NLopt', '2.6.1'),
    ('ParMETIS', '4.0.3'),
]
```

---

Compatible `ParMETIS` can be found in #9886

---

fixed in 6909025